x-pack/plugins/ui_actions_enhanced/public/drilldowns/components/connected_flyout_manage_drilldowns/connected_flyout_manage_drilldowns.tsx

```diff
@@ -4,33 +4,26 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { useEffect, useState, useMemo } from 'react';
+import React, { useState, useMemo } from 'react';
 import { ToastsStart } from 'kibana/public';
-import useMountedState from 'react-use/lib/useMountedState';
-import { intersection } from 'lodash';
+// eslint-disable-next-line no-restricted-imports
+import intersection from 'lodash/intersection';
```

Comment:
please revert (this adds more bytes into bundle)
explanation: https://github.com/elastic/kibana/pull/78156#issue-490969591

Reply:
Done.